config/secrets.yml

```diff
@@ -12,11 +12,18 @@
 
 development:
   secret_key_base: 5091e17fd4c0a3ddacdc134a3926e29c464cdaf0526684912fcd61f89c30680ec5be714f3f66b2c6c7ed6c5384b60eaf31584e10b9cff8e7bdaf9e2d8d6d11ab
-
+  ezid_default_shoulder: doi:10.5072/FK2
+  ezid_user: apitest
+  ezid_password: apitest
```

Comment:
In the ezid-client documentation they instruct that the test user password is not provided and EZID should be contacted. We probably shouldn't be publishing.